bindings/python/README.md

```diff
@@ -0,0 +1,48 @@
+# libiio: Python Bindings
+
+This package contains the python bindings for libiio, a library for interfacing with Linux IIO devices.
+
+libiio is used to interface to the Linux Industrial Input/Output (IIO) Subsystem. The Linux IIO subsystem is intended to provide support for devices that in some sense are analog to digital or digital to analog converters (ADCs, DACs). This includes, but is not limited to ADCs, Accelerometers, Gyros, IMUs, Capacitance to Digital Converters (CDCs), Pressure Sensors, Color, Light and Proximity Sensors, Temperature Sensors, Magnetometers, DACs, DDS (Direct Digital Synthesis), PLLs (Phase Locked Loops), Variable/Programmable Gain Amplifiers (VGA, PGA), and RF transceivers. You can use libiio natively on an embedded Linux target (local mode), or use libiio to communicate remotely to that same target from a host Linux, Windows or MAC over USB or Ethernet or Serial.
+
+[![Build Status](https://travis-ci.org/analogdevicesinc/libiio.svg?branch=master)](https://travis-ci.org/analogdevicesinc/libiio)
+[![PyPI version](https://badge.fury.io/py/libiio.svg)](https://badge.fury.io/py/libiio) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/4bd027bfc5774029a30a9e1cedf5a434)](https://www.codacy.com/app/rgetz/libiio?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=analogdevicesinc/libiio&amp;utm_campaign=Badge_Grade)
+![open bugs](https://img.shields.io/github/issues/analogdevicesinc/libiio.svg)
+
+[[Docs](https://analogdevicesinc.github.io/libiio/v0.19/python/index.html)]
+[[Support](http://ez.analog.com)]
+[[Wiki](https://wiki.analog.com/resources/tools-software/linux-software/libiio)]
+
+### Requirements
+To use these bindings naturally you need the core library they depend upon, libiio. This is not packaged with the pypi release but there are a number of options:
+  - If you want to just use libiio, we suggest using the [latest release](https://github.com/analogdevicesinc/libiio/releases/latest).
+  - If you think you have found a bug in the release, or need a feature which isn't in the release, try the latest **untested** binaries from the master branch and check out the [documentation](https://codedocs.xyz/analogdevicesinc/libiio/) based on the master branch. We provide builds for a few operating systems. If you need something else, we can most likely add that -- just ask.
```

Comment:
https://codedocs.xyz/analogdevicesinc/libiio/ isn't the latest - point to github pages.

Reply:
Solved.